instrumentation-core/spring/spring-webflux-5.0/src/main/java/io/opentelemetry/instrumentation/spring/webflux/client/SpringWebfluxHttpClientTracer.java

```diff
@@ -71,4 +68,18 @@ protected String responseHeader(ClientResponse clientResponse, String name) {
     List<String> headers = clientResponse.headers().header(name);
     return !headers.isEmpty() ? headers.get(0) : null;
   }
+
+  @Override
+  protected Setter<ClientRequest> getSetter() {
```

Comment:
Hmm - I'm sensing something wrong with our API if we have to skip it so many times (thought for broader discussion, not an issue with this PR)

Reply:
Updated the adpater to be httpformat.setter<clientrequest>.

Reply:
@iNikem we could make the `HttpHeadersInjectAdapter` implement `HttpTextFormat.Setter<ClientRequest.Builder>`, and add a third generic type to `HttpClientTracer`, maybe named `HEADERS`? this 3rd generic type might solve similar issue in a couple other places too. i'll log an issue for the `getSetter()` not working as broadly as we would like currently

Reply:
#964